recipes/dataframe/all/test_package/test_package.cpp

```diff
@@ -1,34 +1,490 @@
 #include <DataFrame/DataFrame.h>
+#include <DataFrame/DataFrameFinancialVisitors.h>
```

Comment:
Keep test_package.cpp simple as possible. The idea here is checking the generated artifacts (headers and libraries), not running a full example practical example or an unit test.

Reply:
Done